lib/model/gclEditorModel.js

```diff
@@ -0,0 +1,48 @@
+var createClass = require('./../utilities/createClass');
+var AttrValidators = require('./../utilities/attrValidators');
+var Lang = require('./../utilities/lang');
+var Graph = require('./graph');
+var Parse = require('./../parse/parse');
+var ParseError = require('./../parse/parseError');
+var AutoLayout = require('./../layout/dummyLayout');
+
+var GCLEditorModel = createClass({
+
+  attrs: {
+
+    text: {
+      value: '',
+      validator: Lang.isString,
+      setter: function(val) {
+        var parseResult = Parse(val);
```

Comment:
Recommend using `.after` event handler instead of setter. Setters are supposed to be primarily for massaging the value to be set. A `on` event listener could prevent the value from being set *after* this setter is called.